pandas-stubs/core/series.pyi

```diff
@@ -1320,14 +1320,14 @@ class Series(IndexOpsMixin[S1], NDFrame):
         dtype: CategoryDtypeArg,
         copy: _bool = ...,
         errors: IgnoreRaise = ...,
-    ) -> Series: ...
+    ) -> Series[Any]: ...
```

Comment:
```suggestion
    ) -> Series[CategoricalDtype]: ...
```
If this change is failing a test, then modify the test.

Reply:
Made that change and made one of the test that previously expected the `Series[Any]` fit with this.

Also double checked that pyright strict that not throw any new warnings due to this.